vagrant/etc/ImageMagick-6/policy.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policymap [
+<!ELEMENT policymap (policy)+>
+<!ELEMENT policy (#PCDATA)>
+<!ATTLIST policy domain (delegate|coder|filter|path|resource) #IMPLIED>
+<!ATTLIST policy name CDATA #IMPLIED>
+<!ATTLIST policy rights CDATA #IMPLIED>
+<!ATTLIST policy pattern CDATA #IMPLIED>
+<!ATTLIST policy value CDATA #IMPLIED>
+]>
```

Comment:
Where did this `<!DOCTYPE>` declaration come from? It doesn't match the one listed at https://imagemagick.org/source/policy.xml, linked to from https://imagemagick.org/script/security-policy.php .

Reply:
No idea!